package.json

```diff
@@ -20,6 +20,8 @@
     "cypress": "13.6.1",
     "govuk-frontend": "^4.4.0",
     "jquery": "^3.6.1",
+    "opg-sirius-search-ui": "https://github.com/ministryofjustice/opg-sirius-search-ui",
+    "punycode": "^2.3.1",
```

Comment:
What do we need punycode for here? Is it a dependency of opg-sirius-search-ui?

Reply:
I believe a package in cypress depends on it, when I run `yarn why punycode` I get this output `info This module exists because "cypress#@cypress#request#tough-cookie" depends on it.`